### DISCUSSION TITLE
Disc title tagging

### DISCUSSION THREAD

**Discussion by auranunderscore** [beetbox/beets#4740] [Q&A] [answered]
Hi! I’m struggling with manually tagging albums with disc title (or, rather, move my pre-existing disc title tags into whatever place Beets accepts them). What’s the exact tags that Beets reads for each filetype for the $disctitle? Any particular tagger I should be using for that? (Yes I'm aware beets has manual tagging but a lot of my library already has disc title tags and I don't wanna go through them all over again)

Thanks!

**Comment by jackwilsdon** [comment, marked as answer]
[Answered on Discourse](https://discourse.beets.io/t/disc-title-tagging/2313)
